### PR TITLE
Fixed issues that prevented installation

### PR DESCRIPTION
**Pull request by aschempp**
See the individual commits for each issue.

### CODE REVIEW

File: src/Controller/InstallProjectController.php [hunk 1]
```diff
@@ -110,6 +110,10 @@ public function configureAction(Request $request)
 
         if ($inputData->has('configuration/php_cli')) {
             $tensideConfig->set('php_cli', $inputData->get('configuration/php_cli'));
+        } elseif ('' !== PHP_BINARY && file_exists(PHP_BINARY)) {
```
Comment (Toflar) [inline src/Controller/InstallProjectController.php]: Shouldn't we maybe call the `autoconfig` route first? Is that the purpose of the `autoconfig` route?

Comment (aschempp) [inline src/Controller/InstallProjectController.php]: That would be a question for @discordier 
This fixed my issue of using the wrong PHP binary which resulted in multiple issues (e.g. timezone not set, intl extension missing etc.)

**Comment by discordier** [comment]
I had to split up the commits and altered them, therefore this PR will not get merged.

* The detection of the php binaries has been moved to https://github.com/tenside/core/commit/d1b9f7c310f5ad5d7e3a9a5cbf869949fd597573.
@Toflar is right in his remark in https://github.com/tenside/core-bundle/pull/4#discussion-diff-57125537, the php binary to use shall be obtained via autoconfig and sent in the configuration API call.
* The micro fixes like shellescape removal etc. have been applied in this repo.
* The install task will not be started automatically anymore (btw. it was an initial request by you @aschempp).

The lock file is not completed yet, as the way you changed it opens race conditions. Furthermore, the resulting answer is not correct anymore as the API result will always be "task has been started" even if it has not been started at all (only a cli process has been spawned but that one can run into the lock file problem).

**Comment by aschempp** [comment]
I assume this can be closed?

**Comment by Toflar** [comment]
Yeah, I think so :)

**Comment by discordier** [comment]
Guess so too. :)

**Pull request closed without merging**